R/initialize_modules.R

```diff
@@ -74,45 +74,50 @@ initialize_module <- function(parameters, data, module_name) {
 
   if (module_class_name == "Fleet") {
     module_fields <- setdiff(module_fields, c(
-      "log_expected_index",
-      "proportion_catch_numbers_at_age"
+      "log_index_expected",
+      "log_landings_expected",
+      "index_expected",
+      "landings_expected",
+      "agecomp_expected",
+      "agecomp_proportion",
+      "observed_index_units",
+      "observed_landings_units"
     ))
 
     fleet_types <- get_data(data) |>
       dplyr::filter(name == module_name) |>
       dplyr::pull(type) |>
       unique()
 
-    if ("landings" %in% fleet_types) {
-      module_fields <- setdiff(module_fields, c(
-        "log_q",
-        "random_q",
-        "estimate_q"
-      ))
-    } else {
-      module_fields <- setdiff(module_fields, c(
-        "log_Fmort"
-      ))
-    }
+    # if (!"index" %in% fleet_types) {
+    #   module_fields <- setdiff(module_fields, c(
+    #     "log_q",
+    #     "random_q",
+    #     "estimate_q"
+    #   ))
+    # }
+
+    # if (!"landings" %in% fleet_types) {
+    #   module_fields <- setdiff(module_fields, c(
+    #     "log_Fmort"
+    #   ))
+    # }
```

Comment:
@nathanvaughan-NOAA did you mean to leave this commented out? Can we remove the code?

Reply:
Yep we can delete it, it's not needed.